### ISSUE TITLE
After composing a Toot, VoiceOver focus glitches

### ISSUE THREAD

**Issue by rkingett** [dNitza/mercury-issues#8]
After composing a brand new Toot, after the toot is posted, VoiceOver focus immediately gets pulled to the status bar at the top of the screen. 

**To Reproduce**

1. Start VoiceOver.
2. Compose a new Toot and send the toot.

**Expected behavior**

VoiceOver focus should be back on the timeline, preferably where you were at before composing the toot.

## Device specs.

iPhone SE: 2020.

OS: IOS 13.6.

**Comment by dNitza** [comment]
I think this one should be fixed now :)

It was also happening on other screens to as you pointed out. I have been fixing this wherever I have been noticing it. Thankfully there aren't many screens in the app, so I think I checked them all, but if you notice any other places, let me know.

N.B. I did notice that sometimes an app in the background that turned on location services would cause VoiceOver to focus the location services item in the status bar. I haven't been able to figure out how to prevent this from happening.
